weatherApp.js

```diff
@@ -122,7 +124,9 @@ windButton.addEventListener("click", renderWindContent);
 
 //zapukać do api i połączyć się z nim za pomocą XHR i promisów. Ew async await
 //zrobić listę danych: Miasto(nagłówek), niżej prognoza pogody na dziś, jutro, pojutrze w formacie `data-"rainy"`
-
+const upperFirstLetter = (name) => {
+  return `${name[0].toUpperCase()}${name.slice(1)}`
```

Comment:
You can probably do that with only css:
https://css-tricks.com/almanac/selectors/f/first-letter/

Reply:
Done. It works, thx :)